cdc/cdc/sink/statistics.go

```diff
@@ -41,6 +41,7 @@ func NewStatistics(ctx context.Context, name string, opts map[string]string) *St
 	statistics.metricExecTxnHis = execTxnHistogram.WithLabelValues(statistics.captureAddr, statistics.changefeedID, name)
 	statistics.metricExecBatchHis = execBatchHistogram.WithLabelValues(statistics.captureAddr, statistics.changefeedID, name)
 	statistics.metricExecErrCnt = executionErrorCounter.WithLabelValues(statistics.captureAddr, statistics.changefeedID, name)
+	statistics.metricInvalidKeyCnt = tikvSinkInvalidKeyCountCounter.WithLabelValues(statistics.captureAddr, statistics.changefeedID, name)
```

Comment:
How about reuse the `executionErrorCounter` with `type = name + "invalid_key"` ? Then we don't need to modify the ["tikv_cdc.json"](https://github.com/tikv/migration/blob/main/cdc/metrics/grafana/tikv-cdc.json).

Reply:
Great